databrowser/src/pages/DatasetRawPage.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <ContentArea>
+    <div>
+      Dataset Detail Page (type = {{ $route.params.datasetType }}, ID =
+      {{ $route.params.datasetId }})
+    </div>
+    <div>
+      <DataTable>
+        <template #tableHeader>
+          <TableHeader>Key</TableHeader>
+          <TableHeader>Value</TableHeader>
+        </template>
+        <template #tableBody>
+          <TableLoadingRow v-if="apiResult.isFetching" />
+          <TableRow
+            v-for="(entry, index) in Object.entries(apiResult.data?.data)"
+            v-else
+            :key="index"
+          >
+            <TableCell>{{ entry[0] }}</TableCell>
+            <TableCell class="whitespace-pre-wrap">
+              {{ getEntryValue(entry[1]) }}
+            </TableCell>
+          </TableRow>
+        </template>
```

Comment:
In order to be more flexible I would propose the following API:

```suggestion
        <TableHeader>
          <TableHeader>Key</TableHeader>
          <TableHeader>Value</TableHeader>
        </TableHeader>
        <TableBody>
          <TableLoadingRow v-if="apiResult.isFetching" />
          <TableRow
            v-for="(entry, index) in Object.entries(apiResult.data?.data)"
            v-else
            :key="index"
          >
            <TableCell>{{ entry[0] }}</TableCell>
            <TableCell class="whitespace-pre-wrap">
              {{ getEntryValue(entry[1]) }}
            </TableCell>
          </TableRow>
        </TableBody>
```

Reply:
This would have the benefit, that the table header is not required and you can do whatever you want.

Reply:
Would you have an example of that? I don't quite understand how this API works.